src/sdk/config_fs.js

```diff
@@ -62,6 +67,7 @@ class ConfigFS {
     constructor(config_root, config_root_backend, fs_context) {
         this.config_root = config_root;
         this.config_root_backend = config_root_backend || config.NSFS_NC_CONFIG_DIR_BACKEND;
+        this.config_dir_version = '1.0.0';
```

Comment:
what is the meaning of the version? when and how do we plan to increment it?

Reply:
I added the following comment to our code, but pasting it here as well - 
```
/**
 * config_dir_version is a semver that describes the config directory's version.  
 * config_dir_version is planned to be upgraded when a change that can not be solved only by backward compatibility 
 * and must require a use of an upgrade script.
 * The config directory upgrade script will handle config directory changes of the structure or content of the config files.
 * The upgrade script will run via `noobaa-cli upgrade run command`
 */
```